gpcontrib/arenadata_toolkit/expected/adb_relation_storage_size_test.out

```diff
@@ -12,9 +15,8 @@ DISTRIBUTED BY (a);
 -- Check that toast exists only for "with_toast" tables
 SELECT relname, reltoastrelid != 0 with_toast
 FROM pg_class
-WHERE relname IN ('heap_table_with_toast', 'heap_table_without_toast',
-                  'ao_table_with_toast', 'ao_table_without_toast')
-ORDER BY 1;
+WHERE relname IN ('ao_table_with_toast', 'ao_table_without_toast',
```

Comment:
I see that here only order of table names was changed. I think it is better to revert the order back, so the delta will be smaller.

Reply:
Order was changed for not to do sorting at the query (by using `ORDER BY`). It is one of small optimizations.

Reply:
The `ORDER` is back. Let's revert the order of table names to reduce delta.